src/components/displayData/displayStocks.jsx

```diff
@@ -0,0 +1,90 @@
+import './displayStocks.css';
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { Container, Row, Col } from 'react-bootstrap';
+import { incrementCount, stocksData } from '../../redux/stocks/stockSlice';
+
+export default function DisplayStocks() {
+  const dispatch = useDispatch();
+  const stocks = useSelector((state) => state.stocks.stocks);
+  const [searchStock, setSearchStock] = useState('');
+
+  useEffect(() => {
+    if (!stocks.length) {
+      dispatch(stocksData());
+    }
+  }, [dispatch, stocks.length]);
+
+  const filteredStocks = stocks.filter((stock) => stock.name
+    .toLowerCase().includes(searchStock.toLowerCase()));
+
+  const middleIndex = Math.ceil(filteredStocks.length / 2); // Calculate the middle index
+
+  const handleIncrementViews = (symbol) => {
+    setTimeout(() => {
+      dispatch(incrementCount({ symbol }));
+    });
+  };
+
+  return (
+    <Container>
+      <div className="d-flex justify-content-center mb-3">
+        <input
+          type="text"
+          placeholder="search stock"
+          value={searchStock}
+          onChange={(e) => setSearchStock(e.target.value)}
+        />
+      </div>
+
+      <Row>
+        <Col className="p-0">
+          {/* Render column 1 */}
+          <ul className="list-group">
+            {filteredStocks.slice(0, middleIndex).map((stock, index) => (
+              <li key={stock.symbol} className={`list-group-item p-0 ${index % 2 === 0 ? 'even' : 'odd'}`}>
+                <Link
+                  to={`/detail/${stock.symbol}/${stock.name}/${stock.price}/${stock.change}/${stock.dayLow}/${stock.dayHigh}/${stock.marketCap}/${stock.volume}/${stock.exchange}`}
+                >
+                  <button type="submit" onClick={() => handleIncrementViews(stock.symbol)} className="stocks-btn">
+                    {stock.symbol}
+                    {' '}
+                    -
+                    {stock.price}
+                    {' '}
+                    Views:
+                    {stock.views}
+                  </button>
+                </Link>
+              </li>
+            ))}
+          </ul>
+        </Col>
+
+        <Col className="p-0">
+          {/* Render column 2 */}
+          <ul className="list-group">
+            {filteredStocks.slice(middleIndex).map((stock, index) => (
+              <li key={stock.symbol} className={`list-group-item p-0 ${index % 2 !== 0 ? 'even' : 'odd'}`}>
```

Comment:
- Kindly remove all the borders and radius you've added to follow the design as much as you can:

![Screenshot from 2023-08-13 04-12-44](https://github.com/cancelei/Stock-Exchange-APP/assets/59397159/3bdcedb8-545c-4b12-99d8-3956fcae993b)
![Screenshot from 2023-08-13 04-10-26](https://github.com/cancelei/Stock-Exchange-APP/assets/59397159/48a66398-ce5d-4396-8a3a-35114b112c24)